README.md

```diff
@@ -39,9 +39,10 @@ The name mux stands for "HTTP request multiplexer". Like the standard `http.Serv
 ## Install
 
 With a [correctly configured](https://golang.org/doc/install#testing) Go toolchain:
-
+Replace "latest" with specific version
 ```sh
-go get -u github.com/gorilla/mux
+go install github.com/gorilla/mux@latest
```

Comment:
In this case, maybe the command should be `go get -u github.com/gorilla/mux@latest`

Reply:
Agreed - I'm in favor of staying with `go get` over `go install` here.

Reply:
@jackgris  The `go install` command is not appropriate here.  The command fails with the error `package github.com/gorilla/mux is not a main package`.

The documentation is correct as is.  @AlexVulaj  this PR should be rejected.